packages/cozy-doctypes/src/banking/BankAccount.spec.js

```diff
@@ -49,3 +49,32 @@ describe('deleteDuplicateBankAccountsWithNoOperations', () => {
     expect(res.map(x => x._id)).toEqual(['empty'])
   })
 })
+
+describe('incoherences', () => {
+  const incoherent = {
+    institutionLabel: "Caisse d'épargne",
+    cozyMetadata: {
+      createdByApp: 'boursorama'
+    }
+  }
+
+  const coherent = {
+    institutionLabel: "Caisse d'épargne",
+    cozyMetadata: {
+      createdByApp: 'caissedepargne1'
+    }
+  }
+
+  const unknown = {
+    institutionLabel: 'Unknown bank',
+    cozyMetadata: {
+      createdByApp: 'unknown1337'
+    }
+  }
+
+  it('should detect when slug is not coherent with createdByApp', () => {
```

Comment:
maybe you can write 2 more test without `createdByApp` key and another without `cozyMetadata` key